### PR TITLE
 Metadata: Restructure and merge in capabilities

### PR DESCRIPTION
**Pull request by kba**
- Merges Capabilities into Metadata
- Definition list of all the possible metadata fields
- Definition list of all the possible capabilities

### CODE REVIEW

**Review by wanghaisheng — COMMENTED** [review]
what does " _must_ have one " exactly mean? only one can exist or one at least

File: 1.2/spec.md [hunk 1]
```diff
@@ -802,93 +802,116 @@ of grouped elements in the output and fail with an error message if they cannot
...
+The capability to generate specific properties is given by the prefix `ocrp_...`;
+the important properties are:
+
+<dl dfn-for="ocr-capabilities">
+
+  : <dfn value>ocrp_lang</dfn>
+  :: Capable of generating `lang=` attributes
 
-# Metadata
+  : <dfn value for="ocr-capabilities">ocrp_dir</dfn>
```
Comment (zuphilip) [inline 1.2/spec.md]: Are the rendering of `ocrp_lang` and `ocrp_dir` different by purpose?

**Comment by kba** [comment]
> what does " must have one " exactly mean? only one can exist or one at least

Exactly one. It must not be omitted and must not be repeated.

File: 1.2/spec.md (hunk 1 above)
Comment (kba) [reply 1.2/spec.md]: They should render the same since they express the same information but you're right, it's redundant, I'll correct.

Comment (zuphilip) [reply 1.2/spec.md]: Hm... should we then also update the information abou thow to [reference terms in the specs](https://github.com/kba/hocr-spec/wiki/Reference-terms-within-the-specs) in the wiki, where I copied that information from the previous PR?

Comment (kba) [reply 1.2/spec.md]: It's semantically equivalent to specify a `dfn-for=XYZ` attribute on a container element (in this case the definition list) to specifying `for=XYZ` on the `<dfn>` itself. https://tabatkins.github.io/bikeshed/#dfn-for

Comment (zuphilip) [reply 1.2/spec.md]: Ah, it is nested there. I see. Feel free to add some words in the wiki...

Comment (zuphilip) [reply 1.2/spec.md]: I added such a sentence in the wiki.

**Pull request merged**